src/coreComponents/physicsSolvers/surfaceGeneration/SurfaceGenerator.hpp

```diff
@@ -529,10 +529,7 @@ class SurfaceGenerator : public SolverBase
   /// choice of failure criterion
   integer m_failCriterion=1;
 
-  // solid solver name
-  array1d< string > m_solidMaterialNames;
-
-  localIndex m_solidMaterialFullIndex;
+  array1d< localIndex > m_solidMaterialFullIndex;
```

Comment:
There is a `solidMaterialNamesString` above in this file that may need to be removed as well, I think